src/tezos_interop/fetch_storage.js

```diff
@@ -47,7 +47,7 @@ const error = (error) =>
       shouldObservableSubscriptionRetry: true,
       streamerPollingIntervalMilliseconds: 1000,
       confirmationPollingIntervalSecond: 1,
-      confirmationPollingTimeoutSecond: 4,
+      confirmationPollingTimeoutSecond: 10,
```

Comment:
It is also set to `4` within listen_transaction.js
Regarding your issue, I think you also should go for 10 in the previous file, am I right?